src/content/en/2021/caching.md

```diff
@@ -17,4 +19,205 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
+Caching is a technique that enables the reuse of previously downloaded content. Over the past decade...
+
+
+## CDN Cache adoption
+
+Talk about CDN adoptions, 14% of desktop pages use a Cloudflare CDN.
+
```

Comment:
Are people starting to use CDN's more? Is 14% a lot?

Reply:
In this case we just make a check on what is the most popular CDN. In some way showing 14% for the most famous, shows that there is a large variarity of solutions out there. I wonder how many sites do not use a CDN at all.